source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Pantheon MU Plugin v1.4.4 and WordPress (Composer Managed) update
+published_date: "2024-06-04"
+categories: [wordpress, plugins]
+---
+
+The latest [1.4.4 update](https://github.com/pantheon-systems/pantheon-mu-plugin/releases) of the Pantheon MU Plugin is now available. This update fixes the WordPress Multisite network setup process by by ensuring the correct instructions for WordPress Multisite setup on Pantheon environments are always shown. This update will be included with the next WordPress release. Composer-based WordPress installs can get the update today by running `composer update`.
+
+The latest [WordPress (Composer Managed) (Early Access)](https://github.com/pantheon-upstreams/wordpress-composer-managed) release updates the configuration file filename displayed in the network setup screen to specify `config/application.php` to apply your configuration changes to, rather than the default `wp-config.php` for non-Bedrock Composer installs. This update will be pushed out to all WordPress (Composer Managed)-based installs alongside the MU Plugin update.
+
+The WordPress (Composer Managed) update also includes a Composer script to automatically update the Composer `platform.php` value to match the version of PHP in your `pantheon.yml`. **Note:** [Roots Bedrock](https://roots.io/bedrock/docs/installation/#requirements) does not support below PHP 8.0.
```

Comment:
> The WordPress (Composer Managed) update

I don't think we should do anything different right now, but that phrasing makes me think we should figure out a way to apply version numbers to "WordPress (Composer Managed)". With the vast majority of upstream updates we get to refer to the version numbers of WordPress and Drupal core. Here it's just a repo with new commits in main.

Reply:
I could give it a date-based update, e.g.
```suggestion
The WordPress (Composer Managed) `2024-06-04` update also includes a Composer script to automatically update the Composer `platform.php` value to match the version of PHP in your `pantheon.yml`. **Note:** [Roots Bedrock](https://roots.io/bedrock/docs/installation/#requirements) does not support below PHP 8.0.
```

This is how it's tracked in [the changelog](https://github.com/pantheon-systems/wordpress-composer-managed/blob/default/CHANGELOG.md#2024-06-04)

Reply:
Updated the note to link to the changelog in the Composer Managed upstream.